scripts/lint/metadata.ts

```diff
@@ -32,32 +31,29 @@ export const verifyMetadata = async (
     log(e.message, { file, startLine, content });
   });
 
-  Object.entries(assert).forEach(([k, v]) => {
-    const defacto = metadata[k];
-    if (defacto !== v) {
+  for (const [key, value] of Object.entries(assert)) {
+    const defacto = metadata[key];
+    if (defacto !== value) {
       const line = content
         .split("\n")
-        .findIndex((line) => line.includes(k)) + 1;
+        .findIndex((line) => line.includes(key)) + 1;
 
-      const message = sprintf(
-        "Metadata %s should be %s but is %s",
-        color.bold(k),
-        color.green(v),
-        color.red(String(defacto)),
-      );
+      const message = `Metadata ${color.bold(key)} should be ${
+        color.green(value)
+      } but is ${color.red(String(defacto))}`;
```

Comment:
This looks harder to read than the existing code, I think the existing code is fine here as it is.

Reply:
Hmm. I think it's more readable the other way, but I can revert. Do you want me to revert all such changes of `sprintf`?

Reply:
I think this block is the only place with `sprintf`? 

I think this is less readable since prettier is splitting across multiple lines in an unnatural way, would appreciate if you can revert.

Basically just leave the `const message = ...` as it was before.